docs/docs_src/kafka/testing_with_real_broker/test_with_real_broker.py

```diff
@@ -16,7 +14,8 @@ async def on_output_data(msg: Data):
     async with TestKafkaBroker(broker, with_real=True) as tester:
         await tester.publish(Data(data=0.2), "input_data")
 
-        await asyncio.sleep(2.0)
+        await on_input_data.wait_call(3)
```

Comment:
There is no documentation for wait_call() in Propan. Why do we have this function?

Reply:
It's a testing purposes function added in the FastStream
But, if you are using TestClient, you need no wait any time, publishing working synchronous with consuming this case. So, It is a function helpful in the real broker tests

Reply:
It was not documented anywhere and it is very useful. We have to add it to docs about testing with real broker.